styles/components/_layout.scss

```diff
@@ -244,12 +281,30 @@
   margin-left: $pad;
 }
 
+@mixin colHGutter($colPad) {
```

Comment:
Yes. This mixin should never be used anywhere else, which is why it's in this file.